sentinel-adapter/sentinel-web-servlet/src/main/java/com/alibaba/csp/sentinel/adapter/servlet/CommonFilter.java

```diff
@@ -78,7 +87,7 @@ public void doFilter(ServletRequest request, ServletResponse response, FilterCha
             if (!StringUtil.isEmpty(target)) {
                 // Parse the request origin using registered origin parser.
                 String origin = parseOrigin(sRequest);
-                ContextUtil.enter(WebServletConfig.WEB_SERVLET_CONTEXT_NAME, origin);
+                ContextUtil.enter(webContextUnify ? WebServletConfig.WEB_SERVLET_CONTEXT_NAME : target, origin);
```

Comment:
How about enabling `webContextUnify` by default? In most circumstances, users will not use flow control by web entrance, but removing this could reduce memory footprint.

Reply:
Enabling `webContextUnify` by `registration.addInitParameter(CommonFilter.WEB_CONTEXT_UNIFY, "false");`
Sorry? I do not understand what you mean.
抱歉，宿何老师 我没有get到你的意思

Reply:
Sorry, I'd seen the wrong part. Ignore my comment...